src/averageAges.js

```diff
@@ -14,12 +14,20 @@
  *
  * @return {number}
  */
+
+const reduceCalc = (arr, arrLength) => {
+  return arr.reduce(
+    (start, age) => start + (age.died - age.born), 0) / arrLength.length;
+};
```

Comment:
you don't need 2 params here. you can just work with `arr`
```suggestion
const reduceCalc = (arr) => {
  return arr.reduce((start, age) => start + (age.died - age.born), 0) / arr.length;
};
```